index.html

```diff
@@ -2595,6 +2595,139 @@ <h3>Data Schemas</h3>
 
       </section>
 
+      <section>
+        <h2>External Resource Integrity</h2>
+        <p>
+          When including a link to an external resource in a VC, it is 
+          desirable to know whether the resource that is pointed to is the
+          same at signing time as at verification time.  This applies
+          to cases where there is an external resource that is
+          remotely retrieved as well as to cases where the issuer and/or
+          verifier may have local cached copies of a resource.
+        </p>
+        <p>
+          It is also desirable to know that the contents of the
+          context(s) used in the verifiable credential are the same when
+          used by both the issuer and verifier. 
+        </p>
+        <p>
+          To validate that a resource referenced by a Verifiable Credential is
+          the same at verification time as at issuing time, an implementer
+          MAY include a property named <code>resourceIntegrity</code> that
+          stores an array of objects that describe additional integrity
+          metadata about each resource referenced by the VC. If
+          <code>resourceIntegrity</code>
+          is present, there MUST be an object in the array for each remote
+          resource for each context used in the verifiable credential. 
+        </p>
+        <p>
+          Each object in the
+          <code>resourceIntegrity</code> array MUST contain the following:
+          the [[URL]] to the resource named <code>id</code> and the 
+          <code>integrity</code> information for the resource
+          constructed using the method specified in <a
+          href="https://www.w3.org/TR/SRI/#integrity-metadata">Subresource
+          Integrity</a>.
+        </p>
+        <p>
+          There MUST NOT be more than one object in the
+          <code>resourceIntegrity</code> per <code>id</code>. 
```

Comment:
```suggestion
          There MUST NOT be more than one object in the
          <code>resourceIntegrity</code> per <code>id</code>. 
```

I don't think we need this statement. If there is, and the values are the same, no problem. if there is, and the values are different, then there is a problem (and you'd expect software to kick out a warning/error when they detect it).